webservices/tasks/__init__.py

```diff
@@ -12,19 +13,25 @@
     schedule = {
         'refresh': {
             'task': 'webservices.tasks.refresh.refresh',
-            'schedule': crontab(minute=0, hour=9),
+            'schedule': crontab(minute=0, hour=9), 
```

Comment:
Should we fix this as well not that we have  realized that the times are all UTC?

Reply:
I think this is fine.  We want to refresh data 5am EST (9am UTC)